traffic_light_list:
  -
    id : 1
    pose :
      x : -14.76
      y : 28.84
      z : 0.0